app/views/home/index.html.erb

```diff
@@ -21,17 +21,16 @@
     <%= link_to "ポエムを作成する", new_poem_path, class: "btn btn-primary btn-lg" %>
   </p>
   <ul class="list-group">
-    <% @users.each do |user| %>
+    <% @poems.each do |poem| %>
     <li class="list-group-item pull-left col-md-4">
       <div class="icon-area">
-        <%= image_tag(user.image_url, class: "img-circle", width: 60) %>
+        <%= image_tag(poem.user.image_url, class: "img-circle", width: 60) %>
       </div>
       <div>
-        <% poem = user.poem.order(:updated_at).last %>
-        <%= link_to poem.title, poem %>
+        <%= link_to poem.title.truncate(20), poem %>
       </div>
       <div>
-        <%= user.name %>
+        <%= poem.user.name %>
```

Comment:
__[rails_best_practices]__
* law of demeter [:ref](http://rails-bestpractices.com/posts/2010/07/24/the-law-of-demeter/)